constant/index.ts

```diff
@@ -1,71 +1,13 @@
-export { graphEndpoint, graphEndpointV1 } from "./query/graphEndpoint";
-export {
-  activeVotesKey,
-  balancesQueryKeys,
-  committedVotesForDelegatorKey,
-  committedVotesKey,
-  contentfulDataKey,
-  decryptedVotesKey,
-  delegateToStakerKey,
-  delegatorSetEventForDelegateKey,
-  delegatorSetEventsForDelegatorKey,
-  encryptedVotesKey,
-  hasActiveVotesKey,
-  ignoredRequestToBeDelegateAddressesKey,
-  outstandingRewardsKey,
-  pastVotesKey,
-  receivedRequestsToBeDelegateKey,
-  revealedVotesKey,
-  sentRequestsToBeDelegateKey,
-  stakerDetailsKey,
-  tokenAllowanceKey,
-  unstakeCoolDownKey,
-  unstakedBalanceKey,
-  upcomingVotesKey,
-  userDataKey,
-  voterFromDelegateKey,
-  voteTransactionHashesKey,
-  votingNotUserDependentQueryKeys,
-  votingQueryKeys,
-  votingUserDependentQueryKeys,
-} from "./query/queryKeys";
-export {
-  black,
-  blackOpacity25,
-  blackOpacity50,
-  blackOpacity60,
-  blackOpacity75,
-  green,
-  grey100,
-  grey50,
-  grey500,
-  grey800,
-  loadingSkeletonOpacity10,
-  loadingSkeletonOpacity100,
-  red100,
-  red500,
-  red500Opacity5,
-  red600,
-  white,
-  whiteOpacity10,
-} from "./styles/colors";
-export { bannerHeight, desktopMaxWidth, desktopPanelWidth, headerHeight } from "./styles/containers";
-export {
-  headerLg,
-  headerMd,
-  headerSm,
-  headerXl,
-  headerXs,
-  textFine,
-  textLg,
-  textMd,
-  textSm,
-  textXs,
-} from "./styles/fonts";
-export { shadow1, shadow2, shadow3 } from "./styles/shadows";
-export { discordLink } from "./voting/discordLink";
-export { earlyRequestMagicNumber } from "./voting/earlyRequestMagicNumber";
-export { numPhases, phaseLength, phaseLengthMilliseconds, roundLength } from "./voting/voteTiming";
-export { votingContractAddress, votingTokenContractAddress } from "./web3/addresses";
-export { goerliDeployBlock } from "./web3/deployBlocks";
-export { signingMessage } from "./web3/signingMessage";
+export * from "./misc/errorMessages";
```

Comment:
so much cleaner. the previous was autogenerated by a tool, but this is much better